Source/WebCore/page/ios/EventHandlerIOS.mm

```diff
@@ -813,11 +813,11 @@ static IntPoint adjustAutoscrollDestinationForInsetEdges(IntPoint autoscrollPoin
     auto hitTestedMouseEvent = document->prepareMouseEvent(hitType, documentPoint, syntheticMouseMoveEvent);
 
     auto subframe = dynamicDowncast<LocalFrame>(subframeForHitTestResult(hitTestedMouseEvent));
-    if (subframe && subframe->eventHandler().tryToBeginDragAtPoint(adjustedClientPosition, adjustedGlobalPosition) == DragStartRequestResult::Started)
-        return DragStartRequestResult::Started;
+    if (subframe)
+        return subframe->eventHandler().tryToBeginDragAtPoint(adjustedClientPosition, adjustedGlobalPosition, WTFMove(completionHandler));
```

Comment:
I _think_ this part is probably fine, given that starting a drag over the non-draggable part of the inner iframe in https://whsieh.github.io/examples/draggable-iframe doesn't start a drag in the main frame before this change. Would be good to double check that dragging within the inner frame on that test page still works after your change.